docs/source/reference-core.rst

```diff
@@ -198,17 +198,17 @@ Time and clocks
 
 Every call to :func:`run` has an associated clock.
 
-By default, trio uses an unspecified monotonic clock, but this can be
```

Comment:
This should stay the same – it's talking about the general guarantees we make about the clock, and we still don't guarantee anything except that it has the property of monotonicity. (This is why we can change from `time.monotonic` to `time.perf_counter` :-).)